XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -106,15 +107,24 @@ public override bool Equals(Alert other)
 
         public static void DownloadAndInstallNewClient(ClientUpdateAlert updateAlert, IWin32Window parent)
         {
-            using (var dialog = new WarningDialog(
-                           string.Format(Messages.UPDATE_CLIENT_CONFIRMATION_MESSAGE, BrandManager.BrandConsole),

```

Comment:
We should also remove `UPDATE_CLIENT_CONFIRMATION_MESSAGE_TITLE` and `UPDATE_CLIENT_CONFIRMATION_MESSAGE` from `Messages.resx`. Doesn't seem they're used anywhere else now:
![image](https://user-images.githubusercontent.com/32554698/156570736-5b42007c-e5c7-46ec-b369-77e948cc3ad6.png)

Reply:
sure thing